README.md

```diff
@@ -227,15 +234,20 @@ provides an opportunity for us to experiment with new design principles.
 
 ### How do I use this on Databricks?
 
-Koalas requires Databricks Runtime 5.x or above. For the regular Databricks Runtime, you should upgrade NumPy and pandas to the appropriate versions and then install Koalas. This can be done using the Libraries tab on the cluster UI, or using dbutils in a notebook:
+Koalas requires Databricks Runtime 5.x or above. For the regular Databricks Runtime, you can install Koalas using the Libraries tab on the cluster UI, or using dbutils in a notebook:
 
 ```python
-dbutils.library.installPyPI("numpy", "1.16.2")
-dbutils.library.installPyPI("pandas", "0.24.2")
 dbutils.library.installPyPI("koalas")
```

Comment:
if that's all, looks like we don't need to even mention ML Runtime. Just have one instruction.

Reply:
I tried with Runtime Version 5.3 (without ML), and it worked.
If someone could double-check, it would be better.